openpype/hosts/fusion/plugins/publish/collect_comp_frame_range.py

```diff
@@ -9,14 +9,20 @@ def get_comp_render_range(comp):
     global_start = comp_attrs["COMPN_GlobalStart"]
     global_end = comp_attrs["COMPN_GlobalEnd"]
 
+    frame_data = comp.GetData("openpype_instance")
+    handle_start = frame_data.get("handleStart", 0)
+    handle_end = frame_data.get("handleEnd", 0)
+    frame_start = frame_data.get("frameStart", 0)
+    frame_end = frame_data.get("frameEnd", 0)
+
     # Whenever render ranges are undefined fall back
     # to the comp's global start and end
     if start == -1000000000:
-        start = global_start
+        start = frame_start
     if end == -1000000000:
-        end = global_end
+        end = frame_end
 
-    return start, end, global_start, global_end
+    return start, end, global_start, global_end, handle_start, handle_end
```

Comment:
This feels so strange to change all of a sudden. I'd say this feels like a HUGE change in behavior since suddenly it's not live based on the comp anymore. I don't understand this mixture between using the comp data and not using the comp data. Could you explain?

Reply:
Agree that this was a bit harsh change. Have rewritten it to be `Custom Range` switch instance attribute. This will render what ever Render range of the viewer is set.